spec/watirspec/wait_spec.rb

```diff
@@ -303,6 +322,34 @@ def present?
         expect { element.wait_while(custom: '') }.to_not raise_exception
       end
 
+      it 'accepts keywords and block' do
+        element = browser.div(id: 'foo')
+        browser.a(id: 'hide_foo').click
+        expect { element.wait_while(custom: '', &:present?) }.to_not raise_exception
```

Comment:
`wait_while(custom: '', &:present?)` will wait while `custom` is blank AND while the element is present. Correct?

Reply:
Yes, exactly